laterpay/asset_sources/js/laterpay-backend-shortcode-generator.js

```diff
@@ -0,0 +1,328 @@
+/**
+ * To add TinyMCE drop down for shortcode generator.
+ *
+ * @package laterpay
+ */
+/*global laterpay_shortcode_generator */
+
+'use strict';
+
+var laterpay_shortcode_genrator = {
```

Comment:
Could we update the usage to a similar usage in other files?

For e.g: `laterpay/asset_sources/js/laterpay-backend-advanced.js`

Update usage to use iife and pass jQuery as $ so you could define common things earlier and use them later.